python/paddle/distributed/fleet/data_generator/data_generator.py

```diff
@@ -230,7 +231,9 @@ def local_iter():
 # add more generalized DataGenerator that can adapt user-defined slot
 # for example, [(name, float_list), (name, str_list), (name, int_list)]
 class MultiSlotStringDataGenerator(DataGenerator):
-    def _gen_str(self, line):
+    def _gen_str(
+        self, line: tuple[tuple[str, list[str]]] | list[tuple[str, list[str]]]
```

Comment:
```suggestion
        self, line: tuple[tuple[str, list[str]], ...] | list[tuple[str, list[str]]]
```

用 `Sequence` ，`Sequence[tuple[str, list[str]]]`